pkg/cloud/services/compute/instance.go

```diff
@@ -300,7 +301,7 @@ func (s *Service) createInstanceImpl(eventObject runtime.Object, openStackCluste
 
 	// Wait for volume to become available
 	if volume != nil {
-		err = util.PollImmediate(retryIntervalInstanceStatus, instanceCreateTimeout, func() (bool, error) {
+		err = wait.PollUntilContextTimeout(context.TODO(), retryIntervalInstanceStatus, instanceCreateTimeout, true, func(context.Context) (bool, error) {
```

Comment:
Nice! We should probably start passing contexts around.

Reply:
I am not familiar with this code base at all, so these are just temp fixes.
Would you prefer to keep it as is for this PR so the code gets compiled and you can fix these up in separate PR or else?

Reply:
I think this can be addressed in a separate PR, `context.TODO()` is good here.